docontrol/manifest.json

```
@@ -0,0 +1,68 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "7622c46e-bade-44ce-99e1-a3529bf4fc04",
+  "app_id": "docontrol",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows",
+    "Category::Security",
+    "Category::Incidents",
+    "Category::Automation",
+    "Category::Alerting"
+  ],
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "SaaS Data Security - modernizing DLP and CASB to Secure SaaS Data",
+    "title": "DoControl",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Use DoControl's security automated workflows to create the most sophisticated, flexible and granular business logics with Datadog's log and incident capabilities.",
```

```suggestion
        "caption": "Use DoControl's security automated workflows to create sophisticated, flexible and granular business logic with Datadog's log and incident capabilities.",
```

We try to avoid using marketing-heavy language for integrations listed on the Integrations page.